src/meta/src/barrier/checkpoint/control.rs

```diff
@@ -609,6 +610,9 @@ pub(crate) struct DatabaseCheckpointControl {
     create_mview_tracker: CreateMviewProgressTracker,
 
     metrics: DatabaseCheckpointControlMetrics,
+
```

Comment:
We can store the field in `CreateMviewProgressTracker`, so that the logic can be naturally ported to snapshot backfill. When handling new barriers, we can inspect the specific field in `CreateMviewProgressTracker` rather than this field in `DatabaseCheckpointControl`.

Reply:
Makes sense, thanks for the suggestion

Reply:
https://github.com/risingwavelabs/risingwave/pull/20967/commits/39760bd58bdf2e328aa7928c15818cbe9e771118